Readme.rst

```diff
@@ -1,13 +1,17 @@
-.. image:: https://travis-ci.org/getpelican/pelican-plugins.svg?branch=master
-    :target: https://travis-ci.org/getpelican/pelican-plugins
-
 Pelican Plugins
 ###############
 
-Beginning with version 3.0, Pelican supports plugins. Plugins are a way to add
-features to Pelican without having to directly modify the Pelican core. Starting
-with 3.2, all plugins (including the ones previously in the core) have been
-moved to this repository, so this is the central place for all plugins.
+**Important note:** We are in the process of migrating plugins from this monolithic repository to their own individual repositories under the new `Pelican Plugins`_ organization, a place for plugin authors to collaborate more broadly with Pelican maintainers and other members of the community. The intention is for all the plugins under the new organization to be in the new “namespace plugin” format, which means these plugins can easily be Pip-installed and recognized immediately by Pelican 4.5+ — without having to explicitly enable them.
+
+This transition process will take some time, so we appreciate your patience in the interim. If you would like to help speed up this transition, the following would be very helpful:
+
+* **If you find a plugin here that has not yet been migrated to the new organization**, create a new issue under this repository and communicate which plugin you would like to help migrate, after which a Pelican maintainer will guide you through the process.
+
+* **If you have come here to submit a pull request to add your plugin**, please consider instead moving your plugin to the `Pelican Plugins`_ organization. Create a new issue under this repository with the details of your plugin, after which a Pelican maintainer will guide you through the process.
```

Comment:
No indication whether the owner then joins the organization and whether they get access rights to their plugin repo.

Reply:
Indeed. The only reason said indication isn't here is that I haven't figured out whether to choose one over the other, or to (continue to) make that call on a case-by-case basis. I'll add that indication as soon as I come up with a better answer. 🤷‍♂️